src/controllers/api.js

```diff
@@ -64,7 +65,7 @@ apiController.loadConfig = async function (req) {
 		'cache-buster': meta.config['cache-buster'] || '',
 		topicPostSort: meta.config.topicPostSort || 'oldest_to_newest',
 		categoryTopicSort: meta.config.categoryTopicSort || 'newest_to_oldest',
-		csrf_token: req.uid >= 0 && req.csrfToken && req.csrfToken(),
+		csrf_token: req.uid >= 0 ? generateToken(req) : undefined,
```

Comment:
It looks like the previous behaviour returned boolean `false` instead of `undefined` here, do you mind fixing? It got caught by one of the tests.

Reply:
Yep, noticed the tests and updated.